default-gecko-config

```diff
@@ -15,8 +15,34 @@ else
 ac_add_options --target=arm-linux-androideabi
 fi
 
+# Follow the section 'Bootstrap dependencies' at
+#
+#  https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/Simple_Firefox_for_Android_build
+#
+# for installing the Android SDK and NDK.
+
+ac_add_options --with-android-sdk="$HOME/.mozbuild/android-sdk-linux"
+ac_add_options --with-android-ndk="$HOME/.mozbuild/android-ndk-r11b"
```

Comment:
Reusing the NDK would be great. I tried to use the ndk and prebuilts/ndk directories in the b2g tree. (using --with-android-ndk and --with-android-toolchain). The problems are:

  - their directory structure is very different from the downloaded NDK and the config scripts don't work
  - (for release 11b) the prebuilds/ndk doesn't contain platforms with versions later than 21, while the downloaded ndk r11b supports versions up to 23